### PR TITLE
Update Italian translations

### PR DESCRIPTION
**Pull request by Jackymancs4**
Issue: #

### Requirements

* Filling out the template is required. Any pull request that does not include enough information to be reviewed in a timely manner may be closed at the maintainers' discretion.

- [ ]  issue was opened to discuss proposed changes before starting implementation.
- [x]  during development, `node` version specified in `package.json` was used (ie using [nvm](https://github.com/creationix/nvm)).
- [x]  package versions and package-lock.json were not changed (`npm install --no-save`).
- [x]  app version number was not changed.
- [x]  all new code has tests to ensure against regressions.
- [x] `npm run lint` reports no offenses.
- [x] `npm run test` is error-free.
- [x]  README and CHANGELOG were updated accordingly.

### Description of the Change

Updated Italian translations in app/locales/it.json. Also updated CHANGELOG accordingly.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -19,6 +19,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 - Improved grammar of microbreak ideas
 - display checkbox list vertically in settings screen 3
 - Dutch translations updated
+- Italian translations updated
```
Comment (hovancik) [inline CHANGELOG.md]: You should create new Updated part under Unreleased.

**Comment by hovancik** [comment]
Hi, thanks for the update ;] Only one small thing for changelog ;]

File: CHANGELOG.md (hunk 1 above)
Comment (Jackymancs4) [reply CHANGELOG.md]: Indeed! Fixed

Comment (Jackymancs4) [reply CHANGELOG.md]: Should I have used a subsection like `### Changed`?

**Pull request merged**